infrastructure/server/oidc_clients_fam.tf

```diff
@@ -3,13 +3,17 @@ resource "aws_cognito_user_pool_client" "fam_console_oidc_client" {
   allowed_oauth_flows                  = ["code"]
   allowed_oauth_flows_user_pool_client = "true"
   allowed_oauth_scopes                 = ["openid", "profile", "email"]
-  callback_urls = (concat(var.fam_callback_urls,
+  callback_urls = (concat(
+    var.fam_callback_urls,
     [
+      var.oidc_sso_playground_url,
       "${aws_api_gateway_deployment.fam_api_gateway_deployment.invoke_url}/docs/oauth2-redirect",
       "${aws_api_gateway_stage.admin_management_api_gateway_stage.invoke_url}/docs/oauth2-redirect"
     ]
   ))
-  logout_urls                                   = var.fam_logout_urls
+  logout_urls                                   = concat(
+    var.fam_logout_urls,
+    [var.oidc_sso_playground_url])
```

Comment:
Hi @MCatherine1994 , sorry, I am wondering, does logout_urls need to include `oidc_sso_playground_url`? Does `oidc_sso_playground_url` provide logout?

Reply:
yes they provide logout, but we might don't need to test the logout. It doesn't hurt to include in the logout urls.

Reply:
Cognito logout seems confusing, I forgot if Cognito enforce them or only on callback. Sure, then leave it for now. Thanks Catherine.